examples-zio2/src/main/scala/tofu/example/logging/ZMakeSimpleExample.scala

```diff
@@ -0,0 +1,26 @@
+package tofu.example.logging
+
+import tofu.example.logging.impl.{FooService, SimpleContext}
+import tofu.logging.zlogs.ZLogging
+import zio._
+
+object ZMakeSimpleExample extends ZIOAppDefault {
+
+  val tasks = (1 to 3)
+    .map(i =>
+      for {
+        id <- Random.nextIntBetween(1000, 4000)
+        _  <- ZIO.serviceWithZIO[SimpleContext](_.set(id.toHexString))
+        _  <- ZIO.serviceWithZIO[FooService](_.foo(i))
```

Comment:
Most of the readers will look at this example only from github web UI, so it makes sense to add `expecation` comments here, like 
```suggestion
        _  <- ZIO.serviceWithZIO[FooService](_.foo(i)) //logs something 
```